src/app.py

```diff
@@ -9,13 +13,26 @@ def say_hello_world():
     return {'result': "Flask says Hello World"}
 
 @app.route('/surgeo', methods=["GET"])
-def surgeo():
+def run_surgeo():
     # TO-DO: @KAYLA
     # Build out a public facing API that responds to a get request with the surname and zip code as query arguments. 
     # The API should return the probabilities of each race prediction as a JSON object.
     # Example of return object: {'AAPI': .56, 'Hispanic': .32, ..., 'White': .10}
 
-    pass
+    surname = request.args.get('surname')
+    zipcode = request.args.get('zipcode')
+    surname_series = pd.Series([surname])
+    zip_series = pd.Series([zipcode]) 
```

Comment:
This is really great, clean, descriptive, idiomatic code. It's super straightforward to read and you know when to break stuff up nicely! Great work!